native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/kafka-native/resource-config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "\\Qkafka/kafka-version.properties\\E"
+      }
+    ]
+  },
+  "bundles": [
+    {
+      "name": "MessagesBundle",
```

Comment:
We can remove this `MessagesBundle` from the resource-config, since this is already added by lang runtime.

Reply:
Module tests failed with `not found` exception when this was not added

Reply:
Yes, it should fail since we have not yet updated the lang version in Kafka. After updating the lang version this should be resolved

Reply:
ack

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/658/commits/d1d31f3845479525050dca80778f575e877ef05a